wp-content/themes/missilethreat/assets/_scss/components/_buttons.scss

```diff
@@ -1,27 +1,17 @@
 @use '../abstracts' as *;
 
 .btn {
+  @extend %font-ui-text-md;
   display: inline-block;
-  padding: rem(8) rem(12);
-  // @extend %font-button;
+  padding: rem(14) rem(20);
+  max-height: rem(40px);
```

Comment:
We don't usually add max-height to buttons - is there a reason for this? Also, it should come before `padding` to pass stylelint. :)